tests/assertion-removal.test.ts

```diff
@@ -34,15 +36,15 @@ describe('assertion-removal', () => {
     it('should not contain "assert" in production', async () => {
       const page = await browser.newPage();
       const response = await page.goto(url, { waitUntil: 'networkidle0' });
-      const sourceHtml = await response.text();
+      const sourceHtml = await response?.text();
       expect(sourceHtml).toContain(
         'It rendered without throwing an assertion error',
       );
     });
   });
 
   describe('build-ssr', () => {
-    let server, closeAssetServer;
+    let server: ChildProcess, closeAssetServer: () => void;
```

Comment:
Surprised there isn't a rule against multiple variable declarations. I'm not really fussed about it, but personally I prefer one var per line.

Reply:
I'm surprised too. I thought we had one 🤔. I can't actually find the rule for this, only an old reference to [this](https://jscs-dev.github.io/rule/disallowMultipleVarDecl).

Reply:
Oh, is it just `one-var` with `never`?

Reply:
I believe the rule and config is [`one-var: ["error", "never"]`](https://eslint.org/docs/latest/rules/one-var#never).